Support for pure Rack apps as first class citizens

ezmobius opened the pull request.

trotter commented:
This is awesome on two fronts. One, you added pure Rack support. Two, you actually work there and submitted your update as a pull request. Totally awesome.

Commit:
Add Rack support as a first class citizen framework

Added rack staging pliugin, rack apps require a config.ru

and a Gemfile to run, this is a stake in the ground to make

life easier for rack apps and to enforce best practices

ezmobius commented:
I'm closing this pull request as it had issues. I will resubmit a much better one with proper tests and better formatting
